app/backend-dotnet/Backend.csproj

```
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
```

```suggestion
    <TargetFramework>net7.0</TargetFramework>
```

Consider using `net7.0`. You'd want to reevaluate NuGet package dependencies too and upgrade where appropriate.